src/components/Dropdown/__tests__/driver.js

```diff
@@ -38,6 +38,10 @@ export default class DropdownDriver {
     return this.renderResult.asFragment();
   }
 
+  get dropdownWrapper() {
```

Comment:
I'm not sure this is in use in your current PR changes, so I wonder if you need it

Reply:
Removing it - eventually not needed it